students/kosarzewski_maciej/lesson_00_the_setup/importable_module.py

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env python3
```

Comment:
Lesson 00 should not be a part of this PR. Lesson 00 is already delivered into jedzej/tietopythontraining-basic